code/modules/antagonists/cult/runes.dm

```diff
@@ -524,7 +524,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 /obj/effect/rune/raise_dead/examine(mob/user)
 	. = ..()
 	if(iscultist(user) || user.stat == DEAD)
-		var/revive_number = LAZYLEN(GLOB.sacrificed) - revives_used
+		var/revive_number = (LAZYLEN(GLOB.sacrificed) - revives_used) / 3
```

Comment:
This is incorrect, see L#559
```dm
var/diff = LAZYLEN(GLOB.sacrificed) - revives_used - SOULS_TO_REVIVE
```

Can you refactor that calculation into a common proc please?

Reply:
Ok I'll try to figure out a refactor, but what conflicts with 559? revives_used is dependant on souls to revive, so it retains the 3* factor. The result will always be the positive integer of needed sacrifices.

Reply:
I'm not seeing where the multiplication happens there though?

Reply:
revives_used starts at -revives_required for a freeby so -3
every time the revive succeeds, revives_used is increased by 3, so it will always be a multiple of 3
diff = sacrifices - sacrifices used -3  = sacrifices -3n -3 where n starts at -1 (that is, revives_used starts at -1*revives_required for the freeby)

Reply:
Ahh, in that case, can you refactor `revives_used` then so it's actually the number of revives used and not a multiple and that and `SOULS_TO_REVIVE` please?